src/components/sellerPages/profile/dashboard/edit/EditInfo.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { fetchMyInfo, editMyInfo } from '../../../../../state/actions';
+import { useOktaAuth } from '@okta/okta-react';
+import NavBar from '../../../../common/navBar';
+
+function EditInfo(props) {
+  const history = useHistory();
+  const { authState } = useOktaAuth();
+  const [sellerForm, setSellerForm] = useState({
+    id: `${props.myInfo.id}`,
+    seller_name: `${props.myInfo.seller_name}`,
+    email_address: `${props.myInfo.email_address}`,
+    phone_number: `${props.myInfo.phone_number}`,
+    physical_address: `${props.myInfo.physical_address}`,
+    description: `${props.myInfo.description}`,
+  });
+
+  useEffect(() => {
+    props.fetchMyInfo(authState);
+  }, []);
+
+  function editForm(e) {
+    console.log('sellerform', sellerForm);
+    setSellerForm({
+      ...sellerForm,
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  function cancelEdit(e) {
+    history.go(-1);
+  }
+
+  async function submitEdit(e) {
+    await props.editMyInfo(authState, sellerForm);
+    history.push('/myprofile/myinfo');
+  }
+
+  return (
+    <>
+      <NavBar />
+      <br />
+      <br />
+      <label>
+        Name
+        <input
```

Comment:
You can use antd library from Ant Design for styling input form. In this case we will have consistence in styling. For example:
`<Form.Item
          name="seller_name"
          rules={[
            {
              required: true,
              message: 'Please input an edited seller name',
            },
          ]}
        >
          <Input placeholder="Seller Name" />
        </Form.Item>`

Reply:
Excellent catch